### DISCUSSION TITLE
want some advices

### DISCUSSION THREAD

**Discussion by Unofficial-Life** [zyrouge/symphony#14] [General]
I am trying to make the app compatible with android 8.1, I have one. I just changed the minsdk in [build.gradle] and when I run the workflow it shows an error in signing 

![image](https://user-images.githubusercontent.com/118039878/207780172-9143efed-ead5-434a-9794-c1fe91fcc016.png)
please look at the attached screenshot as I am not a programmer and know nothing about coding I am just doing minimum changes so it can work for me 

I liked the UI when I first looked at it but it's not compatible

**Comment by zyrouge** [comment]
You need to set the env variables for the build command. You could build a debug apk by changing the command to `./gradlew assembleDebug`. I would prefer actually using Android Studio instead of the workflow for testing. Also, I assume there would be undefined behaviour due to `MediaStore` limitations below Android 9. If you could contact me on discord, I could try to lower the sdk version and give test builds to you. I didn't include any version below Android 9 since I don't have a device to test on.
